src/ET_Client.php

```diff
@@ -238,12 +283,6 @@ function __construct($getWSDL = false, $debug = false, $params = null)
             }
 		}
 
-		$context = stream_context_create([
```

Comment:
The $context variable is used on line 248 and the code will fail. Add back the $context variable declaration. This is used for proxy.

Reply:
added back